code/include/playfab_c/PlayFabGlobal_c.h

```diff
@@ -0,0 +1,31 @@
+#pragma once
+
+#include <httpClient/pal.h>
+#include <httpClient/async.h>
+
+/// <summary>
+/// Handle to global state created by PlayFabInitialize. A PlayFabStateHandle will be needed to call login methods.
+/// Cleaned up with PlayFabCleanupAsync.
+/// </summary>
+typedef struct PlayFabGlobalState* PlayFabStateHandle;
+
+/// <summary>
+/// Create PlayFab global state.
+/// </summary>
+/// <param name="stateHandle">Pointer to PlayFabStateHandle.</param>
+/// <returns>Result code for this API operation.</returns>
+HRESULT PlayFabInitialize(
+    _Outptr_ PlayFabStateHandle* stateHandle

```

Comment:
>stateHandle [](start = 33, length = 11)

Why are we exposing the global state handle? If its global state, couldn't we manage the state behind the scenes and return E_PF_NOT_INITIALIZED when the global state isn't initialized?